gulp-core-build-sass/src/SassTask.ts

```diff
@@ -41,8 +47,9 @@ export class SassTask extends GulpTask<ISassTaskConfig> {
     sassMatch: [
       'src/**/*.scss'
     ],
-    useCSSModules: false,
-    dropCssFiles: false
+    treatAllFilesAsCSSModules: false,
```

Comment:
shouldn't we just keep the `useCSSModules` setting name the same so nothing breaks?